Dockerfile

```diff
@@ -71,13 +59,25 @@ ENV CXX $CXX
 RUN . $NVM_DIR/nvm.sh && nvm install $NODE_VERSION && nvm use $NODE_VERSION && \
     npm install -g node-gyp && node-gyp install
 
+# Copy sources
+COPY . .
```

Comment:
I'm probably missing something obvious but why are we copying current dir to current dir?